suite/tests/client-interface/cleancall.dll.c

```diff
@@ -146,6 +216,20 @@ bb_event(void *drcontext, void *tag, instrlist_t *bb, bool for_trace, bool trans
         dr_insert_clean_call(drcontext, bb, add, (void *)cleancall_no_aflags_save, false,
                              0);
         dr_insert_clean_call(drcontext, bb, cmp, (void *)cleancall_aflags_save, false, 0);
+
+#ifdef X86
+        /* Another unrelated test for setting register values.
+         * More specifically, this is a test for setting XMM value.
+         * cleancall to store original XMM value and set new value.
+         * cleancall to check new XMM value and restore to the original value.
+         */
+        dr_insert_clean_call(drcontext, bb, instr, set_xmm, false, 0);
```

Comment:
Could you also please add a sub-test for ymm and zmm if possible. zmm only for #ifdef __AVX512F__
Alternatively just add a comment /* XXX i#1312: Add test for ymm, zmm. */